src/transformers/tape.js

```diff
@@ -78,8 +78,6 @@ const tPropertiesUnsupported = new Set([
     'notDeepLooseEqual',
     'notLooseEqual',
     'notLooseEquals',
-
-    'skip',
```

Comment:
@aaronmcadam I've added a test case on master. https://github.com/skovhus/jest-codemods/commit/b9b4b2d2dc0cc495ced3a6ce1abccd7907aa7ce5#diff-8a42e61f22c2af4312bacd90f87af29aR399

Can you pull that in? Then I think we can merge and release this :)